schemas/mozilla_nimbus_schemas/experiments/experiments.py

```diff
@@ -273,3 +294,50 @@ class NimbusExperiment(BaseModel):
         ),
         default=None,
     )
+
+    @model_validator(mode="after")
+    @classmethod
+    def validate_firefox_labs(cls, data: Self) -> Self:
+        if data.isFirefoxLabsOptIn:
+            if data.firefoxLabsTitle is None:
+                raise ValueError(
+                    "firefoxLabsTitle field cannot be none "
+                    "if isFirefoxLabsOptIn is True"
```

Comment:
how about: `"missing field firefoxLabsTitle (required because isFirefoxLabsOptIn is True)"`